tests/lightning_tensor/test_lightning_tensor.py

```diff
@@ -0,0 +1,75 @@
+# Copyright 2018-2024 Xanadu Quantum Technologies Inc.
+
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+
+#     http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""
+Unit tests for the generic lightning tensor class.
+"""
+
+
+import numpy as np
+import pennylane as qml
+import pytest
+from conftest import LightningDevice  # tested device
+from pennylane.wires import Wires
+
+from pennylane_lightning.lightning_tensor import LightningTensor
+
+if not LightningDevice._new_API:
+    pytest.skip("Exclusive tests for new API. Skipping.", allow_module_level=True)
+
+if LightningDevice._CPP_BINARY_AVAILABLE:
+    pytest.skip("Device doesn't have C++ support yet.", allow_module_level=True)
+
+
+@pytest.mark.parametrize("num_wires", [None, 4])
+@pytest.mark.parametrize("c_dtype", [np.complex64, np.complex128])
+def test_device_name_and_init(num_wires, c_dtype):
+    """Test the class initialization and returned properties."""
+    wires = Wires(range(num_wires)) if num_wires else None
+    dev = LightningTensor(wires=wires, c_dtype=c_dtype)
```

Comment:
```suggestion
    dev = LightningDevice(wires=wires, c_dtype=c_dtype)
```

Reply:
Do you intend on accepting these changes @PietropaoloFrisoni ?

Reply:
We decided to modify the github workflow for this device in a separate PR as it takes more time than we expected